proto/pdpb.proto

```diff
@@ -224,11 +224,25 @@ message ScanRegionsRequest {
     bytes end_key = 4; // end_key is +inf when it is empty.
 }
 
+message Region {
```

Comment:
It seems we can use it for some other responses like `GetRegionResponse`

Reply:
yes. But GetRegion already has pending and down peers fields. And it will need some efforts to keep it compatible.